manifest_feature.yml

```diff
@@ -1,12 +1,16 @@
 ---
-inherit: manifest_base.yml
-host: fec-feature-proxy
+buildpack: staticfile_buildpack
+applications:
+  - name: proxy
+    memory: 64M
+routes:
+  - route: fec-feature-proxy.app.cloud.gov
 services:
   - fec-creds-feature
 env:
   PROXIES: |
     {
       "/": "https://fec-feature-cms.app.cloud.gov",
-      "/data": "https://fec-feature-web.app.cloud.gov",
```

Comment:
Yup, no longer needed!  We don't have a separate `web` app anymore, so no need for this `/data` proxy rule. :-)

Reply:
Ah okay, I see, it is the same bucket as `dev`, which was intended when we set this back up.

Reply:
You know what, we don't even need this file - we don't run the `proxy` app in the `feature` space anymore. :-)

Reply:
@ccostino I'll remove it with a new commit.